Patches.py

```diff
@@ -1445,8 +1445,8 @@ def calculate_traded_flags(world):
     # Patch freestanding items
     if world.settings.shuffle_freestanding_items:
         # Get freestanding item locations
-        actor_override_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'ActorOverride']
-        rupeetower_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'RupeeTower']
+        actor_override_locations = [location for location in world.get_locations() if not location.locked and location.type == 'ActorOverride']
```

Comment:
As we discussed on today's call, this entire block of code can just be removed. It's some dead code from the early potsanity days